platform/mds/mds-web/src/main/resources/webapp/partials/widgets/field-basic.html

```diff
@@ -25,6 +25,9 @@
             <span class="form-hint" ng-class="{hide: uniqueField(field.basic.name)}">
                 {{msg('mds.error.duplicateNames')}}
             </span>
+            <span class="form-hint" ng-class="{hide: isValidJavaFieldName(field.basic.name) || field.basic.name.length === 0}">
```

Comment:
will this error display even before I start editing, since it checks for 0 length (the good old failed before you started issue)?